Bio/FSSP/fssp_rec.py

```diff
@@ -1,4 +1,7 @@
-# A superclass for reading [f]ixed-column type [f]lat-[f]ile records. (e.g.
+'''A superclass for reading [f]ixed-column type [f]lat-[f]ile records.'''
```

Comment:
Got it.

> On Oct 25, 2015, at 5:55 PM, Peter Cock <notifications@github.com> wrote:
> 
> In Bio/FSSP/fssp_rec.py <https://github.com/biopython/biopython/pull/646#discussion_r42950982>:
> 
> > @@ -1,4 +1,7 @@
> > -# A superclass for reading [f]ixed-column type [f]lat-[f]ile records. (e.g.
> > +'''A superclass for reading [f]ixed-column type [f]lat-[f]ile records.'''
> We tend to use triple double quotes in Biopython, rather than triple single quotes.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/biopython/biopython/pull/646/files#r42950982>.
>